src/styles/index.scss

```diff
@@ -1,3 +1,22 @@
-body {
+@import "utils/var";
+@import "blocks/card";
+
+body,
+div,
+p,
+a {
   margin: 0;
+  box-sizing: border-box;
+  font-family: $font-family;
+}
+
+p {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.container {
+  display: flex;
+  width: 200px;
```

Comment:
This wrapper is not necessary. Move `width` property to `card` and remove container